src/js/modules/infragistics.ui.editors.js

```diff
@@ -7787,21 +7787,37 @@
 				spin type="string" Spin buttons are located on the right side of the editor
 			*/
 			buttonType: "none",
-			/* type="number" Gets/Sets delta-value which is used to increment or decrement the editor date on spin actions.
+			/* type="number|object" Gets/Sets delta-value which is used to increment or decrement the editor date on spin actions.
 				When not editing (focused) the delta is applied on the day if available in the input mask or the lowest available period.
 				When in edit mode the time period, where the cursor is positioned, is incremented or decremented with the defined delta value.
-				The value can be only a positive integer number, otherwise it will be set as 1, or in the cases with double or float the the whole part will be taken.
+				If spinDelta is defined as an intereger, then this value it is applied to all time periods - years, days, minutes, etc.

```

Comment:
@Lipata Can you split those into separate descriptions for number and object (enum-like)?